src/components/QueryEditor.tsx

```diff
@@ -34,6 +34,7 @@ const stravaActivityTypeOptions: Array<SelectableValue<StravaActivityType>> = [
   { value: 'Run', label: 'Run' },
   { value: 'Ride', label: 'Ride' },
   { value: 'Other', label: 'Other' },
+  { value: 'Custom', label: 'Custom' },
```

Comment:
Isn't better to convert activity selector to support multi-value selection and add variables to the list?

Reply:
How would you know which variable to put in the list?

Reply:
Usually, it's just a list of created variables from templateSrv, but it might be bad from UX perspective.